arviz/tests/saved_models/cmdstan/example_stan.data.R

```diff
@@ -0,0 +1,5 @@
+x <- 1
```

Comment:
Also just curious, is R required for cmdstan?

Reply:
No. `<-` is just part of Rdump format that cmdstan reads.